SimplenoteUITests/Generic.swift

```diff
@@ -47,6 +47,23 @@ func toggleSwitchIfNeeded(_ switchElement: XCUIElement, _ value: String) {
     switchElement.tap()
 }
 
+func dismissVerifyEmailIfNeeded(using app: XCUIApplication) {
```

Comment:
Since `app` is defined as a global variable in all the files in `SimplenoteUITests` (the approach in `SimplenoteScreenshots` is different), it's not really necessary to have the argument here:

```suggestion
func dismissVerifyEmailIfNeeded() {
```